website/docs/guides/core-to-cloud-2.md

```diff
@@ -0,0 +1,196 @@
+---
+title: 'Move from dbt Core to dbt Cloud: What you need to know'
+id: core-to-cloud-2
+description: "Use this guide to understand the considerations and methods you need to move from dbt Core to dbt Cloud."
+hoverSnippet: "Use this guide to understand the considerations and methods you need to move from dbt Core to dbt Cloud."
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Migration','dbt Core','dbt Cloud']
+keywords: ['dbt Core','dbt Cloud','Migration', 'Move dbt', 'Migrate dbt']
+level: 'Intermediate'
+recently_updated: true
+---
+
+## Introduction
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed platform.
+
+Explore our 3-part-guide series on moving from dbt Core to dbt Cloud. The series is ideal for users aiming for streamlined workflows and enhanced analytics:
```

Comment:
hey @jtcohen6 how does this look to you?  https://docs-getdbt-mnq3cnuh5-dbt-labs.vercel.app/guides/core-to-cloud-2?step=1
<img width="687" alt="Screenshot 2024-04-05 at 13 10 01" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/9a7cf518-2c66-432d-a6cc-1174ca47eb46">

Reply:
I like it! Thank you for incorporating this feedback & running with it :)

In terms of sequence: It feels like "What you need to know" might come _before_ "Get started"?